packages/buildessential.rb

```diff
@@ -64,5 +64,8 @@ class Buildessential < Package
 
   # perl module build ?
   # depends_on 'perl_module_build'
+  
+  # archive utilities
```

Comment:
Move `tar` into `  # compression utilities` part?

Reply:
Is there a compelling reason we need to do this?  I'm in the camp of limiting core packages.